src/sentry/static/sentry/app/views/settings/account/accountClose.jsx

```diff
@@ -116,48 +124,49 @@ class AccountClose extends AsyncView {
           {t('This will permanently remove all associated data for your user')}.
         </TextBlock>
 
-        <TextBlock>
+        <Alert type="error" icon="icon-circle-exclamation">
           <Important>
             {t('Closing your account is permanent and cannot be undone')}!
           </Important>
-        </TextBlock>
-
-        {!!organizations.length && (
-          <TextBlock>
-            {t('If you continue, the following organizations will be removed')}:
-          </TextBlock>
-        )}
-
-        <ul>
-          {organizations.map(({organization, singleOwner}) => {
-            return (
-              <li key={organization.slug}>
-                <label>
-                  <input
-                    style={{marginRight: 6}}
-                    type="checkbox"
-                    value={organization.slug}
-                    onChange={this.handleChange.bind(this, organization, singleOwner)}
-                    name="organizations"
-                    checked={
-                      orgsToRemove === null
-                        ? singleOwner
-                        : orgsToRemove.has(organization.slug)
-                    }
-                    disabled={singleOwner}
-                  />
-                  {organization.name} ({organization.slug})
-                </label>
-              </li>
-            );
-          })}
-        </ul>
-
-        <TextBlock>
-          Ownership will remain with other members if an organization is not deleted.<br />
-          Disabled boxes mean that there is no other owner within the organization so no
-          one else can take ownership.
-        </TextBlock>
+        </Alert>
+
+        <Panel>
+          <PanelHeader>{t('Remove the following organizations')}</PanelHeader>
+          <PanelBody>
+            <PanelAlert type="info">
+              {t(
+                'Ownership will remain with other members if an organization is not deleted.'
+              )}
+              <br />
+              {t(
+                'Disabled boxes mean that there is no other owner within the organization so no one else can take ownership.'
+              )}
+            </PanelAlert>
+
+            {organizations.map(({organization, singleOwner}) => {
+              return (
```

Comment:
nit: no need for the return, this could be unindented a level if the arrow function points at the jsx node to return